contracts/pool_stable/src/tests/swap.rs

```diff
@@ -231,12 +232,20 @@ fn swap_with_high_fee() {
         }
     );
     // 10% fees are deducted from the swap result and sent to fee recipient address
-    let fees = Decimal::percent(10) * output_amount;
+    let fees = convert_u128_to_i128(
+        Decimal256::percent(&env, 10)
+            .mul(
+                &env,
+                &Decimal256::new(&env, convert_i128_to_u128(output_amount)),
```

Comment:
And what is this, why are you trying to convert output amount into the decimal type?

Reply:
so we can multiply - `Decimal256` currently can be multiplied only with `Self`